mappings/net/minecraft/entity/LivingEntity.mapping

```diff
@@ -100,3 +107,4 @@ CLASS ahu net/minecraft/entity/LivingEntity
 	METHOD p setAbsorptionAmount (F)V
 	METHOD v_ isChild ()Z
 	METHOD w_ canClimb ()Z
+	METHOD z onAttacking (Lahl;)V
```

Comment:
onAttack?

Reply:
Would it be confused with `onAttacked` then?

Reply:
onAttacking